README.markdown

```diff
@@ -2724,7 +2724,7 @@ lua_need_request_body
 
 **phase:** *depends on usage*
 
-Due to the stream processing feature of HTTP2, it does not support HTTP2 connection.
+Due to the stream processing feature of HTTP/2 or HTTP/3, this configuration could potentially block the entire request. Therefore, this configuration is effective only when HTTP/2 or HTTP/3 requests send content-length header. For requests with versions lower than HTTP/2, this configuration can still be used without any problems.
```

Comment:
We have discussion about this problem and reached an agreement that enables `read_body` API in the HTTP2 request to contain content-length header 
And why `chunk-transfer` not enabled?  from RFC
```
In https://greenbytes.de/tech/webdav/rfc7540.html#rfc.section.8.1.p.4:

    HTTP/2 uses DATA frames to carry message payloads. The chunked transfer encoding defined in Section 4.1 of [RFC7230] MUST NOT be used in HTTP/2.
```

Reply:
@zhuizhuhaomeng Hi, Can you help us take a look at this PR?